ci/steps.yml

```diff
@@ -35,12 +35,12 @@ steps:
         if [[ "$(SUBARCHIVES)" != "" ]]; then
           opt+=" --archives $(SUBARCHIVES)"
         fi
-        python -m aqt install $(QT_VERSION) $(HOST) $(TARGET) $(ARCH) $opt
+        python -m aqt install-qt $(QT_VERSION) $(HOST) $(TARGET) $(ARCH) $opt
```

Comment:
Was this PR meant to address #306? It changes the names of the commands, but I thought that #306 was supposed to change the syntax of those commands so that QT_VERSION comes after host and target, like this:

```bash
python -m aqt install-qt $(HOST) $(TARGET) $(QT_VERSION) $(ARCH) $opt
```